src/app/dataset/depublication/depublication.component.ts

```diff
@@ -310,11 +355,14 @@ export class DepublicationComponent implements OnDestroy {
           }
         });
       })
-    ) as Observable<MoreResults<RecordPublicationInfo>>;
+    ) as Observable<MoreResults<RecordDepublicationInfo>>;
```

Comment:
According to my IDE this assertion is unnecessary and it does not change the type of the expression.